accounts/abi/bind/v2/backend.go

```diff
@@ -44,9 +44,9 @@ var (
 	// an empty contract behind.
 	ErrNoCodeAfterDeploy = errors.New("no contract code after deployment")
 
-	// Returned by WaitDeployed when the receipt for the transaction hash does not contain
-	// a contract address. This error may indicated that the transaction hash was not a
-	// CREATE transaction.
+	// ErrNoAddressInReceipt is returned by WaitDeployed when the receipt for the
```

Comment:
@rjl493456442 BTW, while it is the normal Go style to start Godoc function comments with the name of the function, it is not required for variables. I usually leave it out because it makes the documentation so much more verbose. When GoDoc renders this block (https://pkg.go.dev/github.com/ethereum/go-ethereum/accounts/abi/bind#pkg-variables) it will just show the code and comments as-is.